dna/plot.py

```diff
@@ -13,3 +18,48 @@ def plot_at_k_scores(model_names: pd.Series, model_scores, model_colors, plot_pa
     plt.legend(loc=0)
     plt.savefig(plot_path)
     plt.clf()
+
+
+def create_distribution_plots(agg_results: dict, output_dir: str):
+    scores_by_dataset = agg_results[0]['test_scores']['scores_by_dataset_id']
```

Comment:
is agg_results a dictionary? Why is it indexed like a list? Are the keys ints?

Reply:
It is a list of dictionaries

Reply:
Yeah good point to change the function description